POST_HERE/app.py

```diff
@@ -0,0 +1,35 @@
+from decouple import config
+from dotenv import load_dotenv
+from flask import Flask, render_template, request, url_for, redirect, jsonify
+from .functions import get_subreddit_info
+import pymongo
+import random
+
+
+load_dotenv()
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route('/')
+    def index():
```

Comment:
Since the app has single or specific  functionality we can use a single route instead of splitting into two routes.